Sources/Plasma/PubUtilLib/plResMgr/plRegistryHelpers.h

```diff
@@ -83,12 +82,8 @@ class plKeyCollector : public plRegistryKeyIterator
     std::set<plKey>& fKeys;
 
 public:
-    plKeyCollector(std::set<plKey>& keys) : fKeys(keys) { }
-    bool EatKey(const plKey& key) override
-    {
-        fKeys.insert(key);
-        return true;
-    }
+    plKeyCollector(std::set<plKey>& keys);
+    bool EatKey(const plKey& key) override;
```

Comment:
We already have the `<set>` include here...

Reply:
Yes, but it looks like I tossed the `plKey` include. But... this is the res manager and it already has a public link against pnKeyedObject, so trying to limit the scope of plKey seems silly here. I'll revert this one.